src/Middlewares/InputFieldMiddlewareInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TheCodingMachine\GraphQLite\Middlewares;
+
+use TheCodingMachine\GraphQLite\InputField;
+use TheCodingMachine\GraphQLite\InputFieldDescriptor;
+
+/**
+ * A middleware use to process annotations when evaluating a input field
+ *
+ * @unstable See https://graphqlite.thecodingmachine.io/docs/semver.html
```

Comment:
What is meant by this annotation? Link is broken.

Reply:
copied from FieldMiddlewareInterface

Reply:
Let's remove it then. The link is dead and doesn't make sense.